src/OpenTelemetry.SemanticConventions/scripts/generate.ps1

```diff
@@ -2,8 +2,8 @@ $SCRIPT_DIR = $PSScriptRoot
 $ROOT_DIR = "${SCRIPT_DIR}/../"
 
 # freeze the spec version to make SemanticAttributes generation reproducible
-$SPEC_VERSION = "1.25.0"
-$GENERATOR_VERSION = "latest"
+$SEMCONV_VERSION = "1.26.0"
```

Comment:
Not sure I follow. What committed version you mean? Once things are committed, nothing is suppose to change or?

Reply:
It will be great to have validation on CI that the code committed by the PR author is correctly generated, without any manual modifications.

I am asking to run script in CI-pipeline and check if there is no differences. Similar what we have for `dotnet format`.

Reply:
If you are fine, we can do it in the separate PR. It will be easier to focus just on this script/pipeline.
IMO it is not hard-blocker as long as you want to provide it. :)

Reply:
Alright I will remove it from here and keep it as a follow up. I created an issue, feel free to assign it to me. https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/2068